.prettierignore

```diff
@@ -0,0 +1 @@
+external/builder/fixtures_esprima/evals.json
```

Comment:
This seems way too specific.
Note that we ignore the same set of files for both [ESLint](https://github.com/mozilla/pdf.js/blob/master/.eslintignore) and [Stylelint](https://github.com/mozilla/pdf.js/blob/master/.stylelintignore), so please use the same list here as well (and don't forget to add a trailing newline at the end).

Reply:
Sorry, but I was suggesting that this file should contain a *verbatim copy* of e.g. https://github.com/mozilla/pdf.js/blob/master/.eslintignore